exercises/run-length-encoding/tests/run-length-encoding.rs

```diff
@@ -0,0 +1,87 @@
+extern crate run_length_encoding as cipher;
+
+// encoding tests
+
+#[test]
+// #[ignore]
+fn test_encode_empty_string() {
+    assert_eq!("", cipher::encode(""));
+}
+
+#[test]
+// #[ignore]
```

Comment:
whereas these should probably be uncommented, right?

I wanted to look for documentation where we say why we have tests ignored, but it is not present. Our documentation is not sufficient in this area. I will have to mention this in #279 .